cadctap/cadctap/core.py

```diff
@@ -483,25 +580,44 @@ def main_app(command='cadc-tap query'):
         help='name of the table (<schema.table>) to load data to')
     load_parser.add_argument(
         'SOURCE', nargs='+',
-        help='source of the data. It can be files or "-" for stdout.'
+        help='source of the data. It can be files or "-" for stdin.'
     )
 
     # handle errors
-    errors = [0]
+#    errors = [0]
```

Comment:
Remove if not used anymore

Reply:
This goes with the commented out function at the end. If we remove this, we should remove that function?

Reply:
Removed.